LICENSES/third_party/forked/golang/LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012 The Go Authors. All rights reserved.
+Copyright (c) 2009 The Go Authors. All rights reserved.
```

Comment:
picked up bugfix to the golang license made in https://github.com/golang/go/commit/d0c1888739ac0d5d0c9f82a4b86945c0351caef6#diff-c693279643b8cd5d248172d9c22cb7cf4ed163a3c98c8a3f69c2717edd3eacb7